### PR TITLE
Update Listing Details page right rail information

### PR DESCRIPTION
**Pull request by pfeeny5**
# Pull Request Template

## Issue

Addresses #348 

- [x] This change addresses the issue in full
- [ ] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description
Removed extra sections from Listing details page right side rail.  This included: Application Status, How to Apply, Public Lottery and What to Expect.
image

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Prototype/POC (not to merge)
- [ ] This change is a refactor/address technical debt
- [ ] This change requires a documentation update
- [ ] This change requires a SQL Script

## How Can This Be Tested/Reviewed?

Compare Listing details page view in desktop to before shot below:
image

- [x] Desktop View
- [ ] Mobile View
- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have updated the changelog to include a description of my changes

### CODE REVIEW

File: sites/public/src/ListingView.tsx [hunk 1]
```diff
@@ -341,31 +328,7 @@ export const ListingView = (props: ListingProps) => {
           <aside className="w-full static md:absolute md:right-0 md:w-1/3 md:top-0 sm:w-2/3 md:ml-2 h-full md:border border-gray-400 bg-white">
             <div className="hidden md:block">
               <ListingUpdated listing={listing} />
-              <ApplicationStatus content={appStatusContent} />
```
Comment (willrlin) [inline sites/public/src/ListingView.tsx:344]: Are there opportunities to remove the appStatusContent variable entirely? I'd imagine we're not using it anymore here.

File: sites/public/src/ListingView.tsx [hunk 2]
```diff
@@ -341,31 +328,7 @@ export const ListingView = (props: ListingProps) => {
...
-              <DownloadLotteryResults event={lotteryResults} />
-              {openHouseEvents && <OpenHouseEvent events={openHouseEvents} />}
```
[the remark this answers is not in the document]
Comment (pfeeny5) [reply sites/public/src/ListingView.tsx:346]: I like this idea. I'll add it back in.

File: sites/public/src/ListingView.tsx (hunk 1 above)
Comment (pfeeny5) [reply sites/public/src/ListingView.tsx:344]: Yes. I've pulled all App and Date items out as well that were defined but never used.

File: sites/public/src/ListingView.tsx [hunk 3]
```diff
@@ -341,31 +328,8 @@ export const ListingView = (props: ListingProps) => {
...
-                  title={t("application.referralApplication.furtherInformation")}
-                />
-              )}
             </div>
-
-            {openHouseEvents && (
-              <div className="mb-2 md:hidden">
-                <OpenHouseEvent events={openHouseEvents} />
-              </div>
-            )}
```
[the remark this answers is not in the document]
Comment (pfeeny5) [reply sites/public/src/ListingView.tsx]: Added back.

**Review by willrlin — APPROVED** [review]

**Pull request merged**